package/deb/vmalert.prerm

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -e
+
+# Reload systemd unit
+systemctl stop vmalert
```

Comment:
for the record the current packaging script, does not respect any debian debhelpers (aka best practices). here `prerm` is called during an `apt remove` command. so disabling the serivce might also be useful ?

Reply:
I would expect removing the systemd unit disables it. Quick check of debhelper generated systemd prerm script from acpid seems to agree, only stop:

```
...
# Automatically added by dh_installsystemd/12.5.2
if [ -d /run/systemd/system ] && [ "$1" = remove ]; then
        deb-systemd-invoke stop 'acpid.service' >/dev/null || true
fi
...
```

Reply:
> I would expect removing the systemd unit disables it. Quick check of debhelper generated systemd prerm script from acpid seems to agree, only stop:
> 
> ```
> ...
> # Automatically added by dh_installsystemd/12.5.2
> if [ -d /run/systemd/system ] && [ "$1" = remove ]; then
>         deb-systemd-invoke stop 'acpid.service' >/dev/null || true
> fi
> ...
> ```

Except "disable" may keep around symlinks that systemd makes to keep track of what services are "wanted". And they those symlinks may become orphaned.

So I'd say stop *and* disable is probably best.